### PR TITLE
19001 Vue 3 upgrade: DocumentDelivery

### PR DESCRIPTION
**Pull request by leodube-aot**
*Issue #:* /bcgov/entity#19001

*Description of changes:*
- Updated DocumentDelivery component to Vue 3

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/relationships-panel/package.json [hunk 1]
```diff
@@ -8,6 +8,6 @@
     "@bcrs-shared-components/enums": "^2.0.0"
   },
   "devDependencies": {
-    "vue-property-decorator": "^9.1.2"
+    "vue-facing-decorator": "^3.0.3"
```
Comment (leodube-aot) [inline src/components/relationships-panel/package.json:11]: Missed this in my RelationshipsPanel PR (`vue-facing-decorator` is imported in the component)

**Review by JazzarKarim — APPROVED** [review]
LGTM! 👍

**Review by ketaki-deodhar — APPROVED** [review]
LGTM!

File: src/components/relationships-panel/package.json (hunk 1 above)
Comment (severinbeauvais) [reply src/components/relationships-panel/package.json:11]: Isn't it one or the other?

Comment (leodube-aot) [reply src/components/relationships-panel/package.json:11]: Yes, we should be using `vue-facing-decorator` for Vue 3. Here I replace `vue-property-decorator` with `vue-facing-decorator`

Comment (severinbeauvais) [reply src/components/relationships-panel/package.json:11]: Ok, gtg

**Pull request merged**